exporter/zipkinexporter/zipkin.go

```diff
@@ -68,15 +68,13 @@ func newZipkinExporter(finalEndpointURI, defaultServiceName string, uploadPeriod
 	return zle, nil
 }
 
-func lookupAttribute(node *commonpb.Node, key string) string {
-	if node == nil {
-		return ""
-	}
-	return node.Attributes[key]
-}
+func zipkinEndpointFromAttributes(
+	attributes *map[string]interface{},
```

Comment:
Why is this a pointer?

Reply:
The idea was just to make clear that the attributes was being changed, as you see it is really not necessary. Anyway, due to the change to not violate the data ownership this is out of the code.